Vagrantfile

```diff
@@ -12,6 +12,11 @@ end
 CLOUD_CONFIG_PATH = File.join(File.dirname(__FILE__), "contrib", "coreos", "user-data")
 CONFIG = File.join(File.dirname(__FILE__), "config.rb")
 
+user_data = File.read(CLOUD_CONFIG_PATH)
```

Comment:
Could you add a comment about why we have to sub in `eth1` here? I had to ask @smothiki and @carmstrong also asked, so others will wonder when they see this code.